client/src/components/Navbar/Navbar.jsx

```diff
@@ -9,22 +10,19 @@ const Navbar = () => {
     <div>
       <AppBar
         position='static'
-        sx={{ background: '#212529', paddingBottom: '15px' }}
+        sx={{ background: '#f1f3f5', paddingBottom: '15px' }}
```

Comment:
this color looks good in the navbar but try a different version and see what is looks best. `sx={{ background: ### '#f1f3f8', paddingBottom: '15px' }}`